edgeai/resnet50-stanford-cars/resnet50-model-upload.ipynb

```diff
@@ -0,0 +1,1838 @@
+      "[DEBUG] GET https://studio.edgeai.zededa.dev/api/v1/user-info\n",
+      "[DEBUG] Headers: {'Authorization': 'Bear...M4Hs'}\n",
+      "[DEBUG] Response: 200 OK\n",
+      "[DEBUG] Response Body: {'user_id': 'celia', 'email': 'celia@company.com', 'name': 'celia norman', 'organization_role': 'systemuser', 'permissions': {'organizations': [{'ids': ['edgeai-studio'], 'permissions': []}], 'catalogs': [{'ids': ['test-catalog-oct1425'], 'permissions': ['models:*']}]}, 'available_catalogs': ['test-catalog-oct1425'], 'all_catalogs': ['test-catalog-oct1425']}\n",
+      "Only one catalog available. Using 'test-catalog-oct1425'.\n",
+      "[DEBUG] POST https://studio.edgeai.zededa.dev/api/v1/auth/catalog-scoped-token\n",
+      "[DEBUG] Payload: {'catalog_id': 'test-catalog-oct1425'}\n",
+      "[DEBUG] Headers: {'Authorization': 'Bear...M4Hs', 'Content-Type': 'application/json'}\n",
+      "[DEBUG] Response: 200 OK\n",
+      "[DEBUG] Response Body: {'access_token': 'eyJh...O6Vc', 'token_type': '***', 'expires_in': 3600, 'catalog_id': 'test-catalog-oct1425', 'user_permissions': ['models:*']}\n",
+      "\n",
+      "Getting MinIO credentials...\n",
+      "[DEBUG] GET https://studio.edgeai.zededa.dev/api/v1/minio/access\n",
+      "[DEBUG] Headers: {'Authorization': 'Bear...O6Vc', 'X-Catalog-ID': 'test-catalog-oct1425'}\n",
+      "[DEBUG] Response: 200 OK\n",
+      "[DEBUG] Response Body: {'access_key': 'cata...9okx', 'secret_key': 'hj6x...seix', 'endpoint_url': 'https://minio.edgeai.zededa.dev', 'bucket': 'mlflow-test-catalog-oct1425', 'expires_in': 3600, 'mlflow_tracking_uri': 'https://studio.edgeai.zededa.dev'}\n",
+      "\n",
+      "Login completed successfully.\n",
+      "\n",
+      "[DEBUG] Response: 200 OK\n",
+      "[DEBUG] Response Body: {'access_token': 'eyJh...O6Vc', 'token_type': '***', 'expires_in': 3600, 'catalog_id': 'test-catalog-oct1425', 'user_permissions': ['models:*']}\n",
+      "\n",
+      "Getting MinIO credentials...\n",
+      "[DEBUG] GET https://studio.edgeai.zededa.dev/api/v1/minio/access\n",
+      "[DEBUG] Headers: {'Authorization': 'Bear...O6Vc', 'X-Catalog-ID': 'test-catalog-oct1425'}\n",
+      "[DEBUG] Response: 200 OK\n",
+      "[DEBUG] Response Body: {'access_key': 'cata...9okx', 'secret_key': 'hj6x...seix', 'endpoint_url': 'https://minio.edgeai.zededa.dev', 'bucket': 'mlflow-test-catalog-oct1425', 'expires_in': 3600, 'mlflow_tracking_uri': 'https://studio.edgeai.zededa.dev'}\n",
+      "\n",
+      "Login completed successfully.\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "from zededa_edgeai_sdk.client import ZededaEdgeAIClient\n",
+    "client = ZededaEdgeAIClient(\n",
+    "    debug=True\n",
+    ")\n",
+    "credentials = client.login(\n",
+    "'test-catalog-oct1425',\n",
+    "email='celia@company.com',\n",
```

Comment:
can you please change the username and password strings, to something else..
like "your-username-here", "your-password-here"